File: Interface Gráfica - Tela de Criação de Herói/TelaCriacaoHeroi.java
import javax.swing.JFrame;
import javax.swing.JLabel;
import javax.swing.JTextField;
import javax.swing.JButton;
import javax.swing.JPanel;
import java.awt.BorderLayout;
import java.awt.GridLayout;

public class TelaCriacaoHeroi extends JFrame {

    private JTextField campoNome;
    private JTextField campoVida;
    private JTextField campoForça;
    private JButton botaoCriar;

    public TelaCriacaoHeroi() {
        super("IF Quest");

        setSize(600, 250);
        setDefaultCloseOperation(EXIT_ON_CLOSE);
        setLayout(new BorderLayout());

        JLabel titulo = new JLabel(
            "Criação de Herói",
            JLabel.CENTER
        );

        add(titulo, BorderLayout.NORTH);

        JPanel painel = new JPanel(
            new GridLayout(3, 2, 5, 5)
        );

        JLabel rotuloNome = new JLabel("Nome:");
        campoNome = new JTextField(5);

        JLabel rotuloVida = new JLabel("Vida:");
        campoVida = new JTextField(5);

        JLabel rotuloForça = new JLabel("Força:");
        campoForça = new JTextField(15);

        painel.add(rotuloNome);
        painel.add(campoNome);
        painel.add(rotuloVida);
        painel.add(campoVida);
        painel.add(rotuloForça);
        painel.add(campoForça);

        add(painel, BorderLayout.CENTER);

        botaoCriar = new JButton("Criar Herói");
        botaoCriar.setEnabled(false);

        add(botaoCriar, BorderLayout.SOUTH);

        titulo.setText("Criação de Herói - IF Quest");
    }

    public static void main(String[] args) {
        TelaCriacaoHeroi tela = new TelaCriacaoHeroi();
        tela.setVisible(true);
    }
}

File: Interface Gráfica - Tela de Criação de Herói/TelaCriacaoHeroi.py
import tkinter as tk


class TelaCriacaoHeroi(tk.Tk):

    def __init__(self):
        super().__init__()

        self.title("IF Quest")
        self.geometry("600x300")

        self.titulo = tk.Label(
            self,
            text="Criação de Herói"
        )
        self.titulo.pack(side="top")

        painel = tk.Frame(self)

        rotulo_nome = tk.Label(
            painel,
            text="Nome:"
        )
        rotulo_nome.grid(row=0, column=0)

        self.campo_nome = tk.Entry(
            painel,
            width=15
        )
        self.campo_nome.grid(row=0, column=1)

        rotulo_vida = tk.Label(
            painel,
            text="Vida:"
        )
        rotulo_vida.grid(row=1, column=0)

        self.campo_vida = tk.Entry(
            painel,
            width=15
        )
        self.campo_vida.grid(row=1, column=1)

        rotulo_forca = tk.Label(
            painel,
            text="Força:"
        )
        rotulo_forca.grid(row=2, column=0)

        self.campo_classe = tk.Entry(
            painel,
            width=15
        )
        self.campo_classe.grid(row=2, column=1)

        painel.pack(expand=True)

        self.botao_criar = tk.Button(
            self,
            text="Criar Herói",
            state="disabled"
        )
        self.botao_criar.pack(side="bottom")

        self.titulo.config(
            text="Criação de Herói - IF Quest"
        )


if __name__ == "__main__":
    tela = TelaCriacaoHeroi()
    tela.mainloop()